### PR TITLE
Stackdriver sink: refactor processing metrics

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Extract looping over unlabeled metrics to a method and put computed metrics in a separate metric set.

cc @piosz

### CODE REVIEW

**Comment by aleksandra-malinowska** [comment]
@k8s-bot test this

**Comment by aleksandra-malinowska** [comment]
@k8s-bot test this

File: metrics/sinks/stackdriver/stackdriver.go [hunk 1]
```diff
@@ -154,18 +166,20 @@ func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
...
-			}
+		timeseries = append(timeseries, computedTimeseries...)
+
+		for _, ts := range timeseries {
+			req.TimeSeries = append(req.TimeSeries, ts)
 			if len(req.TimeSeries) >= maxTimeseriesPerRequest {
 				sink.sendRequest(req)
 				req = getReq()
 			}
+
```
Comment (piosz) [inline metrics/sinks/stackdriver/stackdriver.go]: remove this line

**Review by piosz — APPROVED** [review]
just a nit

**Comment by piosz** [comment]
/lgtm
feel free to merge on green

**Pull request merged**